packages/web/src/server.tsx

```diff
@@ -54,31 +57,35 @@ function getScripts(statsStr: string) {
 }
 
 app.get('*', (req, res) => {
-  const html = ReactDOMServer.renderToString(
-    <Provider store={store}>
-      <StaticRouter location={req.url}>
-        <App locationChangeHandler={() => {}} />
-      </StaticRouter>
-    </Provider>
-  );
+  try {
+    const html = ReactDOMServer.renderToString(
+      <Provider store={store}>
+        <StaticRouter location={req.url}>
+          <App locationChangeHandler={() => {}} />
+        </StaticRouter>
+      </Provider>
+    );
 
-  const helmet = Helmet.renderStatic();
+    const helmet = Helmet.renderStatic();
 
-  readFile(path.join(__dirname, 'static', 'stats.json'), { encoding: 'utf-8' }, (err, stats) => {
-    if (err) {
-      res.status(404).send();
-      return;
-    }
+    readFile(path.join(__dirname, 'static', 'stats.json'), { encoding: 'utf-8' }, (err, stats) => {
+      if (err) {
+        res.status(404).send();
+        return;
+      }
 
-    const { js, css } = getScripts(stats);
+      const { js, css } = getScripts(stats);
 
-    res.send(
-      '<!doctype html>' +
-        ReactDOMServer.renderToString(
-          <Layout js={js} css={css} head={helmet} env={getClientVars()} html={html} />
-        )
-    );
-  });
+      res.send(
+        '<!doctype html>' +
+          ReactDOMServer.renderToString(
+            <Layout js={js} css={css} head={helmet} env={getClientVars()} html={html} />
+          )
+      );
+    });
+  } catch (e) {
+    logger.error(e);
```

Comment:
Did you intend to just swallow all of the errors here? IMO, we should still maintain proper 404 / 500 error codes.

Reply:
I guess it was related to my hot fixes, since the `trpc` doesn't output errors in a console. I was wondering if we can make a single error handler for `trpc` like express error middleware

Reply:
This is not about api, it's about web. 
Added status(500) here. Probably we should pass the status code from inside components and set a proper status code when sending a response.